AnaProd/tasks.py

```diff
@@ -13,6 +13,10 @@
 from FLAF.run_tools.law_customizations import Task, HTCondorWorkflow, copy_param, get_param_value
 from FLAF.Common.Utilities import SerializeObjectToString
 from FLAF.AnaProd.anaCacheProducer import addAnaCaches
+from FLAF.RunKit.law_wlcg import WLCGFileSystem
+
+def get_fs_private(fs_name_private=None, fs_nanoAOD_default=None):
```

Comment:
`get_fs_private` logic should be moved to Setup in order to reuse the same logic as for other FS creation. E.g. what if path is just local path and not at WLCG? What if there are multiple sources etc.